### PR TITLE
382 - Make TitForTat verbose

### PR DESCRIPTION
**Pull request by drvinceknight**
This addresses #382 and make TitForTat verbose.

### CODE REVIEW

File: axelrod/strategies/titfortat.py [hunk 1]
```diff
@@ -4,8 +4,16 @@
 
 
 class TitForTat(Player):
-    """A player starts by cooperating and then mimics previous move by opponent."""
+    """
+    A player starts by cooperating and then mimics previous move by
```
Comment (marcharper) [inline axelrod/strategies/titfortat.py]: Can we say "mimics the previous action of the opponent." ?

**Comment by marcharper** [comment]
:+1:

File: axelrod/strategies/titfortat.py (hunk 1 above)
Comment (drvinceknight) [inline axelrod/strategies/titfortat.py]: Yes that's indeed what it should be. Away from a computer right now but
will make that change (and also change in the docs).

**Comment by drvinceknight** [comment]
@langner jumped the gun a bit, I just need to push the slight doc change. Will throw that straight to master... (If this annoys anyone just let me now and I'll be sure to not do again.)

**Comment by langner** [comment]
> @langner jumped the gun a bit, I just need to push the slight doc change. Will throw that straight to master... (If this annoys anyone just let me now and I'll be sure to not do again.)

Oops. Sorry about that.

**Comment by drvinceknight** [comment]
No problemo :+1: :smile:

**Pull request merged**